new/LIT6942SalamLakiGalila.xml

```diff
@@ -5,8 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">Salām lakki ʾo-hagara Galilā</title>
-                <title xml:lang="en" corresp="#t1">Peace be upon you, O land of Galilee!</title>
+                <title xml:lang="gez" xml:id="t1" type="normalized">Salām lakki ʾo-hagara Galilā</title>
```

Comment:
what I meant was
either, if we use the incipit as the title, we must have
```
                <title xml:lang="gez" xml:id="t1">ሰላም፡ ለኪ፡ ኦሃገረ፡ ገሊላ፡</title>
                <title xml:lang="gez" corresp="#t1" type="normalized">Salām laki ʾo-hagara Galilā</title>
                <title xml:lang="en" corresp="#t1">Peace be upon you, O land of Galilee!</title>
```
or, 
even better,  keeping Strelcyn's label
` <title xml:lang="en">Salām to places connected with the life of Jesus</title>`

Reply:
OK. I put Strelcyn title as t1 and the incipit-based title as t2 with fidal and translation.